yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/YamlTestExtension.java

```diff
@@ -52,11 +56,13 @@
 public class YamlTestExtension implements TestTemplateInvocationContextProvider, BeforeAllCallback, AfterAllCallback {
     private static final Logger logger = LogManager.getLogger(YamlTestExtension.class);
     private List<YamlTestConfig> testConfigs;
+    private List<YamlTestConfig> maintainConfigs;
 
     @Override
     public void beforeAll(final ExtensionContext context) throws Exception {
         if (Boolean.parseBoolean(System.getProperty("tests.runQuick", "false"))) {
             testConfigs = List.of(new EmbeddedConfig());
+            maintainConfigs = List.of();
```

Comment:
If we wanted to rewrite all of the metrics, is there a way of doing that besides, like, setting the `testConfigs` to one of the maintain configs, and then running the tests? It seems like there's a fairly straightforward way in this design to update a single test or a single test file, but not all of the tests.

Reply:
You can annotate the test class which would rewrite the metrics file for every test in that class.

Reply:
I get that. I was asking if there's a way to get it correct all tests across all test files. In particular, I was thinking it would be a good idea to have one that doesn't require updating any of the Java files, as that makes it more difficult to accidentally commit the code in correction-mode